darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -1020,7 +1067,7 @@ def test_static_cov_accuracy(self):
             train_series_no_cov[0][: (60 - period)],
             train_series_no_cov[1][:60],
         ]
-        model_no_static_cov = RandomForest(lags=period // 2, bootstrap=False)
+        model_no_static_cov = RandomForest(lags=period // 2, bootstrap=True)
```

Comment:
I noticed here that bootstrap was set to True for the model_no_static_cov but was left to False for the subsequent model_static_cov, it won't be fair to compare the two. 
Furthermore, in all those tests I was only able to make RandomForest perform better, in general, when bootstrap was set to False, I would assume it is better to leave both at False. WDYT?

Reply:
Good catch - I was playing around with `bootstrap` when I was debugging. It should be set back to `bootstrap=False` as it was in the original test.